drawing/tools/custom/Vector.js

```diff
@@ -352,7 +351,11 @@ if(0 && dojox.drawing.defaults.zAxisEnabled){
 					}
 				},this);
 			};
-			dojo.connect(this, "onRenderStencil", this, function(){ if(this.zSelected){ this.zDeselect(this.zSelected)}});
+			dojo.connect(this, "onRenderStencil", this, function(){ 
```

Comment:
I'd like to get this fix in now, since the fix has been sitting there for four years.